docs/content/install/shared-team-instance.md

```diff
@@ -30,9 +30,9 @@ docker run -d -v $ATHENS_STORAGE:/var/lib/athens \
 
 **PowerShell**
 ```PowerShell
-set $env:ATHENS_STORAGE = "athens-storage"
+set $env:ATHENS_STORAGE = "C:\athens-storage"
```

Comment:
If we use `.\athens-storage` I am hoping that will workaround permission issues _and_ the home directory is shared by default so that docker for windows won't prompt.

What do you think?

Reply:
I changed my mind. This works and people can improve this over time. 😀